File: BACKEND/PickleballCourtBookingSystem/PickleballCourtBookingSystem.Core/Entities/CourtTimeBooking.cs
namespace PickleballCourtBookingSystem.Core.Entities
{
    public class CourtTimeBooking
    {
        public Guid? Id { get; set; }
        public Guid? BookingId { get; set; }
        public Guid? CourtTimeSlotId { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/maxholmes/androidapp/data/model/CourtTimeBooking.kt
package com.maxholmes.androidapp.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class CourtTimeBooking(
    var id: String,
    var bookingId: String,
    var courtTimeSlotId: String
) : Parcelable